buildpipeline/DotNet-CoreClr-Trusted-Windows-x86.json

```diff
@@ -227,7 +227,7 @@
       }
     },
     {
-      "enabled": true,
+      "enabled": false,
```

Comment:
Can we please have a tracking issue to re-enable this when everything is back to normal?